CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# Contributing to the Self Hosted Show Wiki
+
+
+## Objectives of This Wiki
+
+In order to contribute well, you need to understand what the objectives are behind this wiki.
+
+What we are trying to create:
+- A compilation of tools, tips and tricks for self-hosting specific applications.
+- A place where beginners and experts can learn about how to quickly, efficiently and securely deploy applications on their own server(s).
+- Snippets that people can quickly reference in deploying applications.
+- A hub for quality documentation (to the extent possible).
+
+What we are not trying to do:
+- Be another large list of self-hosted tools. See [awesome-selfhosted](https://github.com/awesome-selfhosted/awesome-selfhosted) for a comprehensive list of self-hosted applications.
+- Be a hub for script kitties. People should _learn_ from documentation here not just come here to copy/paste. Snippets are important but should be documented properly.
+- Insecure, sloppy or any undesirable systems administration practices such as:
+  - Executing remote code by `curl`-ing an untrusted script and piping it to `sudo bash`: `curl https://example.com/awesomescript.sh | sudo bash`
+  - Installing additional bloat within a container
+
+
+We want to encourage the usage of Devops tooling such as config management, containerisation and automation. This will improve the ease of use for all and increase code reuse amongst the community ultimately leading to an easier time for all.
+
+## Directory Layout:
+
+The `docs` folder in the root of the repository is where wiki content should be placed.
+
+The following example directory layout should be adhered to when contributing:
+
+`docs/category/application/index.md`
+
+Additional, more specific articles relating to that same topic should be named differently:
+
+`docs/category/application/specific-topic.md` 
+
+For example, if I wanted to add a brand new, generic article for Home Assistant, my layout would be this:
+
+`docs/home-automation/home-assistant/index.md`
+
+
+For an additional article that is more specific I could name it like so:
+
+`docs/home-automation/home-assistant/docker-deployment.md`
+
+
+Any images embedded in an article should be placed within a folder called `images` within the same directory:
+
+`docs/home-automation/home-assistant/images/example.png`
+
+
+Any other relevant resources (scripts, snippets, etc.) pertaining to the application should be placed within the same folder the relevant application is using.
+
+
+## Build locally
+
+- Clone the repository and `cd` into the directory:
+
+```bash
+git clone https://github.com/selfhostedshow/wiki ~/wiki &&\
+	cd ~/wiki
+```
+
+- Make changes as necessary.
+
+- Start development server on http://localhost:8000:
+
+```bash
+docker run --rm -it -p 8000:8000 -v ${PWD}:/docs squidfunk/mkdocs-material
+```
```

Comment:
We will need a line saying something what is docker and how to install it, it could link to an external site or this wiki

Reply:
That's a good point. I will add a link to install Docker.

Reply:
This is resolved with a5383656